src/SonarQube.Client/Models/SonarQubeIssue.cs

```diff
@@ -74,6 +77,9 @@ public class SonarQubeIssue
         public IReadOnlyList<IssueFlow> Flows { get; }
 
         public string Context { get; set; }
+
+        public string CleanCodeAttribute { get; }
+        public Dictionary<string, string> DefaultImpacts { get; set; }
```

Comment:
String is inconvenient to work with up the stack, see SonarQubeIssueSeverity enum for example